how-to-extend/adapter-starterkit/build.gradle

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2018 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+  id 'io.spring.dependency-management' version '1.0.5.RELEASE'
+  id 'java'
+}
+allprojects {
+  repositories {
+    mavenLocal()
+    jcenter()
+    maven { url 'https://plugins.gradle.org/m2/' }
+    maven { url "https://oss.sonatype.org/content/groups/staging/" }
+    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+  }
+}
+
+defaultTasks = ['build']
+description = 'Data Source Adapter Starter-kit'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+tasks.withType(JavaCompile) {
+  options.encoding = 'UTF-8'
+  options.compilerArgs << '-Xlint:deprecation'
+  options.incremental = true
+}
+
+dependencyManagement {
+  imports {
+    mavenBom "io.knotx:knotx-dependencies:${project.version}"
```

Comment:
I think that
`${project.version}`
should be
`${knotx.version}` (e.g. defined in starterkit gradle.properties)